getting-started/installation-windows-azure.md

```diff
@@ -0,0 +1,30 @@
+## Installing from Azure Database for PostgreSQL
+
+TimescaleDB is currently available on [Azure Database for PostgreSQL][azure-postgresql] versions 9.6 and 10. To install TimescaleDB, you need to include it in the server’s shared preloaded libraries. A change to PostgreSQL’s shared preloaded libraries requires a restart to take effect.
+
+*Note: TimescaleDB is only available in version 1.1 from Azure Database for PostgreSQL*
```

Comment:
Sorry! Small nit: Only TimescaleDB v1.1.1 is available on Azure Database for PostgreSQL?